src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -117,6 +122,27 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
             }
         }
 
+        public List<int> GetPortsToIgnoreFromAnnotations(V1ServiceList serviceNameSpace) {
+            var portsToIgnore = new List<int>();
+            List<V1Service> portsList = serviceNameSpace.Items.Where(item => item.Metadata?.Annotations?.ContainsKey(ServiceAnnotations) ?? false).ToList();
+            portsList.ForEach(port => {
+                if (port.Metadata?.Annotations?.TryGetValue(ServiceAnnotations, out string ports) ?? false) {
+                    if (ports.Length > 0) {
+                        try {
+                            // ports e.g. "460, 543"
+                            var ignorePorts = ports.Split(",").Select(p => int.Parse(p));

```

Comment:
Hi Vidya we can still make this one liner avoiding a local variable, if it makes things complex, please ignore it. 

```suggestion
                            portsToIgnore =ports.Splt(",")?.Select(Int32::Parse)?.toList()

```